src/CSS/AboutUs.module.css

```diff
@@ -0,0 +1,109 @@
+.body {
+    padding-top: 0px;
+    background-color: #ffffff;
+  }
+  
+.AboutUs_heading__3Gzv2{
+    letter-spacing: 0;
+    font-size: 2.3rem;
+    font-weight: 390;
+    font-family: "Poppins",sans-serif;
+    text-align: center;
+    margin-bottom: 50px;
+}
+    
+.AboutUs_About__3C3xP{
+    letter-spacing: 0;
+    color: #000;
+    margin: 10px;
+    font-size: 50px;
+    font-weight: 500;
+}
+
+.AboutUs_Us__Embxm{
+    letter-spacing: 0;
+    color: #008dc8;
+    font-size: 50px;
+    font-weight: 500;
+}
+
+.AboutUs_AboutImage__11ku5{ margin-top:15% }
```

Comment:
those class names ... did you manually append hashes(`3Gzv2` etc) after the name?

Reply:
Not actually I saw it in the styles from the site mentioned in the repo i.e. https://girlcodeit.com. If you want me to remove those I can do it. Let me know what should be my next task.
Thanks!

Reply:
Okay. You can remove those extra hashes. It will be generated automatically at build time by webpack.

Reply:
Sure will do that ASAP.

Reply:
@abdus I've made the required changes. You can review them now.
Thanks!